Update Who-Is-Using-Orleans.html

jkonecki opened the pull request:
Added MaxADR to the list of companies using Orleans

JillHeaden commented:
@sergeybykov - in light of this, should I put this page back in the 2.0 documents?

sergeybykov commented:
@jkonecki Thank you for this! There are a couple of problems though.

1. We deleted the page because it became rather misleading due to the fact that a very small subset of users were willing to add them to the list. Need to check why we still have the html page.

2. We moved doc sources to the docs branch, and that's what doc PRs need to target.

jkonecki commented:
No worries. If you don't maintain this page anymore you can abandon the PR.

On Tue, 30 Oct 2018, 17:37 Sergey Bykov, <notifications@github.com> wrote:

> @jkonecki <https://github.com/jkonecki> Thank you for this! There are a
> couple of problems though.
>
>    1.
>
>    We deleted the page because it became rather misleading due to the
>    fact that a very small subset of users were willing to add them to the
>    list. Need to check why we still have the html page.
>    2.
>
>    We moved doc sources to the docs branch, and that's what doc PRs need
>    to target.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/orleans/pull/5139#issuecomment-434396708>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABsWjyU_7pSbSywHXSEMfwSBf8YyzuC4ks5uqI5jgaJpZM4X8DKo>
> .
>
